mvn -P allTests fails with a lot of "Actually, there were zero interactions with this mock"

**Describe the bug**
Slow tests fail with lots of warnings form mockito. Bug introduced in commit e2dbd31. Bug is in tests, not user code.

**To Reproduce**
Steps to reproduce the behavior:
1. `git checkout master`
2. `mvn clean package; mvn test -P allTests`

**Expected behavior**
Build and tests complete successfully.

```
-> at org.bonej.wrapperPlugins.CommonWrapperTests.testNonBinaryImageCancelsPlugin(CommonWrapperTests.java:181)
Actually, there were zero interactions with this mock.

[ERROR]   SurfaceFractionWrapperTest.testNullImageCancelsSurfaceFraction:94 
Wanted but not invoked:
userInterface.dialogPrompt(
    <any string>,
    <any string>,
    <any>,
    <any>
);
-> at org.bonej.wrapperPlugins.CommonWrapperTests.testNullImageCancelsPlugin(CommonWrapperTests.java:114)
Actually, there were zero interactions with this mock.

[INFO] 
[ERROR] Tests run: 147, Failures: 21, Errors: 0, Skipped: 0
[INFO] 
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for BoneJ Parent POM 7.0.14-SNAPSHOT:
[INFO] 
[INFO] BoneJ Parent POM ................................... SUCCESS [  1.392 s]
[INFO] BoneJ legacy utilities ............................. SUCCESS [  5.523 s]
[INFO] BoneJ legacy plugins ............................... SUCCESS [02:01 min]
[INFO] BoneJ2 Ops ......................................... SUCCESS [01:14 min]
[INFO] BoneJ2 utilities ................................... SUCCESS [  9.484 s]
[INFO] BoneJ2 wrapper plugins ............................. FAILURE [01:10 min]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  04:43 min
[INFO] Finished at: 2022-08-26T09:02:55+08:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.22.2:test (default-test) on project bonej-plugins: There are test failures.
[ERROR] 
[ERROR] Please refer to /home/mdoube/git/pom-bonej/Modern/wrapperPlugins/target/surefire-reports for the individual test results.
[ERROR] Please refer to dump files (if any exist) [date].dump, [date]-jvmRun[N].dump and [date].dumpstream.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :bonej-plugins
```

--- a/Modern/wrapperPlugins/src/test/java/org/bonej/wrapperPlugins/AbstractWrapperTest.java
+++ b/Modern/wrapperPlugins/src/test/java/org/bonej/wrapperPlugins/AbstractWrapperTest.java
@@ -96,6 +96,7 @@ public abstract class AbstractWrapperTest {
 
     @Before
     public void setup() {
+    	  imageJ.ui().setHeadless(false);
         imageJ.ui().setDefaultUI(MOCK_UI);
         doNothing().when(MOCK_REPORTER).reportEvent(anyString());
     }

